### DISCUSSION TITLE
Custom Contact Items

### DISCUSSION THREAD

**Discussion by acoluzzi** [cotes2020/jekyll-theme-chirpy#1722] [Ideas]
### Description

I'd like to share what I did in my [website](https://www.thebytessmith.dev/) branched from this theme.

I've added a type of contact item [here](https://github.com/coluzziandrea/thebytessmith/blob/main/_includes/sidebar.html#L58) to be able to write custom icons in the sidebar, like the following (taken from [here](https://github.com/coluzziandrea/thebytessmith/blob/main/_data/contact.yml)):

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/12852000/5a467f4f-6b0d-44a4-a049-efd9d0681de1)

<details>
  <summary>Screenshot</summary>
  
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/12852000/c73bef53-f26d-4fbf-be0c-e3276383def8)
  
</details>

What do you think? If it is something useful I can prepare a PR  👍🏻

**Comment by huanyushi** [comment]
The author has actually set up this function, see [here](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_includes/sidebar.html#L69-L70).

**Comment by acoluzzi** [comment]
Oh thank you @huanyushi ! I'll close this
